### ISSUE TITLE
Container will not start with latest dev release

### ISSUE THREAD

**Issue by tim0901** [haugene/docker-transmission-openvpn#2242]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [X] I have tried using dev branch

### Docker run config used
```
transmission-openvpn:
                container_name: transmission_openvpn
                image: haugene/transmission-openvpn:dev 
                restart: always
                cap_add:
                    - NET_ADMIN
                volumes:
                    - ~/data/transmission-home:/config/transmission-home
                environment:
                    - PUID=1001
                    - GUID=1001
                    - OPENVPN_PROVIDER=WINDSCRIBE
                    - OPENVPN_CONFIG=${OPENVPN_CONFIG}
                    - OPENVPN_USERNAME=${OPENVPN_USERNAME}
                    - OPENVPN_PASSWORD=${OPENVPN_PASSWORD}
                    - OPENVPN_REMAP_USR1="0"
                    - LOCAL_NETWORK=192.168.0.0/16
                    - TZ=Europe/London
                logging:
                    driver: json-file
                    options:
                        max-size: 10m
                networks:
                  static-network-vpn:
                    ipv4_address: 172.20.0.3
                  letsencrypt_proxy-tier:
```
### Current Behavior

When using the current dev release, transmission fails to find .ovpn scripts
```
Starting container with revision: edb73e83d1d46d74a325b2ceaeab628d6c456508
Creating TUN device /dev/net/tun
Using OpenVPN provider: WINDSCRIBE
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for WINDSCRIBE. Defaulting to external config
Supplied config Paris-Jardin-udp.ovpn could not be found.
Your options for this provider are:
```
I don't have login details for other providers, but changing the entry for OPENVPN_PROVIDER did not result in anything appearing below "Your options for this provider are:" and therefore I believe this is not a provider-specific issue.

### Expected Behavior

The container should start as expected

### How have you tried to solve the problem?

Rolling back to the previous release (from 6 days ago) fixes the issue, as does switching to the stable branch.

### Log output

Starting container with revision: edb73e83d1d46d74a325b2ceaeab628d6c456508
Creating TUN device /dev/net/tun
Using OpenVPN provider: WINDSCRIBE
Running with VPN_CONFIG_SOURCE auto
No bundled config script found for WINDSCRIBE. Defaulting to external config
Supplied config Paris-Jardin-udp.ovpn could not be found.
Your options for this provider are:

### HW/SW Environment

```markdown
- OS: Pi OS 64bit Bullseye
- Docker: 20.10.14
```

**Comment by Gerben321** [comment]
Same here after updating.

**Comment by pkishino** [comment]
should be fixed in [51e8a02]
